content/scripts/build.js

```diff
@@ -462,37 +469,33 @@ class Builder {
       const folder = titleData.file;
       if (titleData.source !== source.filepath) {
         allPrerequisiteFlaws.push(
-          prerequisite.createFlaw(
-            `${uri} transcludes ${prerequisiteUri}, which is from a different source`
+          fileInfo.updateFlaw(
+            prerequisite.createFlaw(
+              `${uri} transcludes ${prerequisiteUri}, which is from a different source`
+            )
           )
         );
         continue;
       }
-      const { metadata, rawHtml, fileInfo } = Document.read(
-        source.filepath,
-        folder,
-        false,
-        this.allWikiHistory
-      );
+      const {
+        metadata: prerequisiteMetadata,
+        rawHtml: prerequisiteRawHtml,
+        fileInfo: prerequisiteFileInfo,
```

Comment:
This wasn't necessary, but I did it only to hopefully make it clearer and prevent confusion.